runtime/common/xwalk_content_client.cc

```diff
@@ -1,3 +1,4 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
```

Comment:
I just updated the commit msg, and please check this comment https://github.com/crosswalk-project/crosswalk/pull/2888/files#r25746765.